internal/artifact/uploader.go

```diff
@@ -371,28 +373,30 @@ func (a *Uploader) createUploader() (_ uploader, err error) {
 	case a.conf.Destination == "":
 		a.logger.Info("Uploading to default Buildkite artifact storage")
 		return NewFormUploader(a.logger, FormUploaderConfig{
-			DebugHTTP: a.conf.DebugHTTP,
+			DebugHTTP:    a.conf.DebugHTTP,
+			TraceHTTP:    a.conf.TraceHTTP,
+			DisableHTTP2: a.conf.DisableHTTP2,
 		}), nil
 
 	case strings.HasPrefix(a.conf.Destination, "s3://"):
 		dest = "Amazon S3"
 		return NewS3Uploader(a.logger, S3UploaderConfig{
 			Destination: a.conf.Destination,
-			DebugHTTP:   a.conf.DebugHTTP,
```

Comment:
It seems like this option never did anything?

Reply:
Yep, set and never used.